path_credentials_test.go

```diff
@@ -58,4 +59,4 @@ func TestAcceptanceUserToken(t *testing.T) {
 	t.Run("renew user token cred", acceptanceTestEnv.RenewToken)
 	t.Run("revoke user token cred", acceptanceTestEnv.RevokeToken)
 	// t.Run("cleanup user tokens", acceptanceTestEnv.CleanupUserTokens)
```

Comment:
why is this test commented out?

Reply:
It's not required. The revoke token does the part